snow/snowtest/snowtest.go

```diff
@@ -28,3 +44,40 @@ func ConsensusContext() *snow.ConsensusContext {
 		VertexAcceptor:      noOpAcceptor{},
 	}
 }
+
+func Context(tb testing.TB, chainID ids.ID) *snow.Context {
+	require := require.New(tb)
+
+	ctx := snow.DefaultContextTest()
+
+	ctx.NetworkID = constants.UnitTestID
+	ctx.SubnetID = constants.PrimaryNetworkID
+	ctx.ChainID = chainID
+	ctx.XChainID = XChainID
+	ctx.CChainID = CChainID
+	ctx.AVAXAssetID = ids.GenerateTestID()
```

Comment:
Why are the chains created once and shared the the assetID is created new each time?

Reply:
We normally over-write the AvaxAssetID in tests if we need it but there isn't really a reason to re-generate it every time. Hoisted to a pkg var